functions/twilio.js

```diff
@@ -56,6 +56,52 @@ module.exports = {
     }
   },
 
+  /**
+   * make it possible to pass different auth tokens for different twilio projects
+   */
+  middlewareByProject: (authToken, req, res, next) => {
```

Comment:
instead of messing with the existing function which is used in so many places/important, i added the token argument to this new one to be reused in case we need to again and use for now for the vax support twilio project. happy to do something different though idk if this makes sense!